Tests/SourceKitLSPAPITests/SourceKitLSPAPITests.swift

```diff
@@ -238,18 +243,20 @@ extension SourceKitLSPAPI.BuildDescription {
         for targetName: String,
         graph: ModulesGraph,
         partialArguments: [String],
+        otherFiles: Int,
         isPartOfRootPackage: Bool,
         destination: BuildParameters.Destination = .target
     ) throws -> Bool {
         let target = try XCTUnwrap(graph.module(for: targetName))
         let buildTarget = try XCTUnwrap(self.getBuildTarget(for: target, destination: destination))
 
-        guard let file = buildTarget.sources.first else {
-            XCTFail("build target \(targetName) contains no files")
+        XCTAssertEqual(buildTarget.others.count, otherFiles, "build target \(targetName) contains an incorrect number of other files")
```

Comment:
This new assertion only covers `others`, what about `ignored` and `resources`?

Reply:
Good point. I've updated the test to check all the file types